SQL/0000-00-00-schema.sql

```diff
@@ -275,6 +275,7 @@ CREATE TABLE `flag` (
   `Test_name` varchar(255) NOT NULL default '',
   `CommentID` varchar(255) NOT NULL default '',
   `Data_entry` enum('In Progress','Complete') default NULL,
+  `Data_entry_completion_status` enum('Incomplete','Complete') NOT NULL default 'Incomplete',
```

Comment:
People have always gotten confused between the Data entry flag and the DECS status, maybe this would be a good opportunity to rename the column to something that more accurately describes what it's for and doesn't sound like a different user facing column?

Reply:
@driusan agreed... suggestions ?

Cause as far as I see it... it seems like it's current name is exactly what it does

Reply:
I don't know, the flag is used to determine if you can click the "Data Entry" button in the frontend, right? Maybe something like `Can_mark_data_entry_complete`? Though to be honest I'm not a fan of that name either.

Reply:
this flag is used to determine if all required fields are completed.. that being the case enables/disables a bunch of control panel stuff

how about a boolean `required_elements_completed` Y/N?

Reply:
I like `required_elements_completed`

Reply:
LORIS Meeting Discussion: go forward with Rida's suggestion.